extensions/Worker.Extensions.CosmosDB/src/Config/ICosmosDBServiceFactory.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Cosmos;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    internal interface ICosmosDBServiceFactory
+    {
+        CosmosClient CreateService(string connectionString, CosmosClientOptions cosmosClientOptions);
```

Comment:
nit: the implementation has this parameter as `connectionName`. Can you make them consistent? Whichever is a more suitable name (I think `connectionName` is more appropriate).